modules/game_htmlsample/htmlsample.css

```
@@ -18,7 +24,123 @@
+.examplePreview {
+  position: absolute;
+  top: 28;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  border: 1px #8a8a8a;
+  background-color: #ffffff;
+  overflow: scroll;
+}
+
+.examplePreview window,
```

Stylelint (17.4.0)

[error] Unexpected unknown type selector "window" (selector-type-no-unknown)

(selector-type-no-unknown)